libs/model/package.json

```diff
@@ -73,7 +73,8 @@
     "db-all": "chmod u+x scripts/db-all.sh && ./scripts/db-all.sh",
     "dump-db": "chmod u+x scripts/dump-db.sh && ./scripts/dump-db.sh",
     "reset-db": "chmod u+x scripts/reset-db.sh && ./scripts/reset-db.sh",
-    "cancel-active-contests": "tsx scripts/cancel-active-contests.ts"
+    "cancel-active-contests": "tsx scripts/cancel-active-contests.ts",
+    "create-branch": "chmod u+x scripts/create-neon-branch.sh && ./scripts/create-neon-branch.sh"
```

Comment:
create-neon-branch instead to be more explicit?

Reply:
I had it as that and then shortened it but I can put it back. This isn't intended to stay for very long anyways